### DISCUSSION TITLE
Inconsistency between dnsx output on same subdomain list

### DISCUSSION THREAD

**Discussion by ko2sec** [projectdiscovery/dnsx#121] [Q&A] [answered]
Hi, when I run dnsx on same list I get inconsistent output. I have tested it on my VPS at digitalocean.
dnsx version is 1.0.6 but it was behaving same at 1.0.1 version before I updated it.

![image](https://user-images.githubusercontent.com/38421562/131924256-4d2ee8e2-6479-445f-b767-53c62301a43e.png)

Speedtest result of VPS:

<img width="537" alt="Screen Shot 2021-09-03 at 01 29 26" src="https://user-images.githubusercontent.com/38421562/131924393-3eafae8c-cbe4-4cea-aa61-2f87732560ef.png">

**Comment by Mzack9999** [comment, marked as answer]
@ko2sec Thanks for opening this issue. In general, there might be slight variation depending on input list size and resolvers QoS. How many items were present in the `all-resolved.txt` file?

**Comment by ko2sec** [comment]
There are nearly ~2000 domains in the `all-resolved.txt` file. I will try to debug it with different resolvers, QoS makes sense.
